gammapy/cube/fit.py

```diff
@@ -1,13 +1,20 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
+import logging
 import numpy as np
 from astropy.utils import lazyproperty
 import astropy.units as u
 from ..utils.fitting import Fit, Parameters
-from ..stats import cash
+from ..stats import cash, cstat
 from ..maps import Map, MapAxis
+from .models import SkyModel, SkyModels
 
 __all__ = ["MapEvaluator", "MapDataset"]
 
+log = logging.getLogger(__name__)
+
+
+UPDATE_THRESHOLD = 0.25 * u.deg
```

Comment:
Is it really safe to have a global `update_threshold`?

Reply:
I would expect the criterion to depend on the size of the support and the expected variation scale of the IRFs.

Reply:
In the few test I did it seemed to work well, but I agree it might not be safe enough in general. I'll change it...

Reply:
I replaced the fixed threshold by `.evaluation_radius` and added an additional `CUTOUT_MARGIN`.